modules/gitops-creating-an-application-by-using-the-oc-tool.adoc

```diff
@@ -47,3 +47,10 @@ endif::cluster[]
 ----
 $ oc get application -n openshift-gitops
 ----
+
+. Add a label to the namespace your application is deployed in so that the Argo CD instance in the `openshift-gitops` namespace can manage it:
```

Comment:
```suggestion
. Add a label to the namespace where your application is deployed. This way, the Argo CD instance in the `openshift-gitops` namespace can manage your <thing>:
```
Does separating this into two sentences make it easier to read? What does "it" refer to?

Reply:
the "it" is referring to the namespace where your application is deployed in... so how would I say that without sounding repetitive?